source/instructors/setupcourses/assignment-settings/penalties.rst

```diff
@@ -10,6 +10,8 @@ You can add penalties to the assignment if students don't complete it within the
 
 Penalties are not cumulative. Your student will receive the penalty associated with their submission date.
```

Comment:
This statement is no longer accurate. The student score will be reduced by the specified percent each time the penalty period elapses.